Python_new_2023/Section 8/test1.py

```diff
@@ -0,0 +1,11 @@
+run = True
```

Comment:
Please explain why you need to add this file. 
It would be great if you edit the PR description accordingly.

Reply:
In my case it is not necessary, I can use - While True
But at the time of creating the code, it seemed to me the right solution